crates/uv/src/commands/python/install.rs

```diff
@@ -727,7 +739,10 @@ fn create_bin_links(
     errors: &mut Vec<(InstallErrorKind, PythonInstallationKey, Error)>,
     preview: PreviewMode,
 ) {
-    let targets = if (default || (is_default_install && !reinstall))
+    // TODO(zanieb): We want more feedback on the `is_default_install` behavior before stabilizing
+    // it. In particular, it may be confusing because it does not apply when versions are loaded
+    // from a `.python-version` file.
+    let targets = if (default || (is_default_install && preview.is_enabled()))
```

Comment:
We _could_ stabilize the `is_default_install` behavior (i.e., `uv python install` gets you `python` / `python3` / `python3.13` while `uv python install 3.13` only gets you `python3.13`) separately from the `--default` flag. The default install behavior is really helpful for beginners, who expect `uv python install` to result in an available `python` executable, but I'm not sure how common it is for people to run it without a version request.